spec/ics-2-consensus-verification/README.md

```diff
@@ -0,0 +1,310 @@
+---
+
+ics: 2
+title: Consensus Verification
+stage: draft
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-04-02
+
+---
+
+## Synopsis
+
+This standard specifies the properties that consensus algorithms of chains implementing IBC are
+expected to satisfy. The properties are needed for efficient and safe verification in the higher
+level protocol abstractions. The algorithm which uses these properties to verify substates of
+another chain is referred to as a "light client".
+
+## Specification
+
+### Motivation
+
+`FullNode`s are procedures running a `Consensus`. Given a `([Transaction], Commit)`, a
```

Comment:
This reads like definitions, not motivation.

This section should answer: Why do we need light clients? What properties do they provide to other parts of IBC?

Move these two paragraphs to definitions.

Reply:
Removed two paragraphs, the fullnode-lightclient correspondence is now described in VERIFIER.md